lib/web/sessions.go

```diff
@@ -698,6 +698,9 @@ func (s *sessionCache) invalidateSession(ctx *SessionContext) error {
 	if err != nil && !trace.IsNotFound(err) {
 		return trace.Wrap(err)
 	}
+	if err := clt.DeleteUserAppSessions(context.TODO(), &proto.DeleteUserAppSessionsRequest{Username: ctx.user}); err != nil {
```

Comment:
Should we also do this when user does `tsh app logout` in the CLI?

Reply:
Looking at the code, it does something similar by removing the application sessions one by one.